cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -936,6 +797,28 @@ function init() {
   return true;
 }
 
+function startLoading() {
+  chart.startLoading();
+  let item;
+  for ( let i = 0, len = dataLoadedDomList.length; i < len; i++ ) {
```

Comment:
Use dom tools here.

Reply:
Are we sure this is necessary in this case? I'd rather not create the external dependency on fastdom if not needed. I ran fastdom-strict through webpack and wrapped the errors it throws in try…catch blocks ([downloadable here](https://gist.github.com/anselmbradford/d02263ae3d917d5b7c245b98f6cf8f72)). I added it to the base-common.html as:

```
<script src="/static/js/fastdom-strict.js">
```

And that outputs hundreds of reports, but the majority are from SVGElements in highcharts presumably.
I haven't been able to find the instances noted here. They may be there, but since the highcharts dependency isn't run through fastdom, is it worth it for these two class list sets?

This is the log:
[fastdom-output.log](https://github.com/cfpb/cfgov-refresh/files/1938859/fastdom-output.log)

Reply:
I'm not sure, let's revisit at a later time.